lib/Cake/Model/Datasource/Database/Postgres.php

```diff
@@ -198,82 +198,138 @@ public function describe($model) {
 		$fields = parent::describe($table);
 		$this->_sequenceMap[$table] = array();
 		$cols = null;
-
 		if ($fields === null) {
-			$cols = $this->_execute(
-				"SELECT DISTINCT table_schema AS schema, column_name AS name, data_type AS type, is_nullable AS null,
-					column_default AS default, ordinal_position AS position, character_maximum_length AS char_length,
-					character_octet_length AS oct_length FROM information_schema.columns
-				WHERE table_name = ? AND table_schema = ?  ORDER BY position",
-				array($table, $this->config['schema'])
-			);
+            try {
```

Comment:
I apologize for the confusion between the bot and what I'm saying, but 2.x uses tabs. I'll try and get the bot disabled for this branch as it is applying the 3.x style guide to this branch which is wrong.